tests/firewall.rs

```diff
@@ -34,7 +34,7 @@ async fn firewall_allow() {
     let yaml = "
 on_read:
   - action: ALLOW
-    source: 127.0.0.1/32
+    source: ::/32
```

Comment:
Replace `::/32` with `::1/128` as that's the common address for ipv6 loopback :)